src/core/sagas/versions.js

```diff
@@ -57,7 +57,7 @@ export function* fetchVersions({
     };
     const versions = yield call(getVersions, params);
 
-    yield put(loadVersions({ slug, versions }));
+    yield put(loadVersions({ slug, versions: versions.results }));
```

Comment:
This saga is missing some type hints for the API response object. Here is an example of how Flow would have caught this:

```diff
diff --git a/src/core/api/versions.js b/src/core/api/versions.js
index bf8aa702d..020165760 100644
--- a/src/core/api/versions.js
+++ b/src/core/api/versions.js
@@ -36,13 +36,13 @@ export type GetVersionsParams = {|
   slug: string,
 |};
 
+export type GetVersionsResponse = PaginatedApiResponse<ExternalAddonVersionType>;
+
 export const getVersions = ({
   api,
   slug,
   ...params
-}: GetVersionsParams): Promise<
-  PaginatedApiResponse<ExternalAddonVersionType>,
-> => {
+}: GetVersionsParams): Promise<GetVersionsResponse> => {
   invariant(slug, 'slug is required');
 
   return callApi({
diff --git a/src/core/sagas/versions.js b/src/core/sagas/versions.js
index bad930fd2..c1ac4f23d 100644
--- a/src/core/sagas/versions.js
+++ b/src/core/sagas/versions.js
@@ -9,7 +9,11 @@ import {
 } from 'core/reducers/versions';
 import log from 'core/logger';
 import { createErrorHandler, getState } from 'core/sagas/utils';
-import type { GetVersionParams, GetVersionsParams } from 'core/api/versions';
+import type {
+  GetVersionParams,
+  GetVersionsParams,
+  GetVersionsResponse,
+} from 'core/api/versions';
 import type {
   FetchVersionAction,
   FetchVersionsAction,
@@ -55,9 +59,10 @@ export function* fetchVersions({
       page,
       slug,
     };
-    const versions = yield call(getVersions, params);
+    const versions: GetVersionsResponse = yield call(getVersions, params);
 
-    yield put(loadVersions({ slug, versions: versions.results }));
+    // yield put(loadVersions({ slug, versions: versions.results }));
+    yield put(loadVersions({ slug, versions }));
   } catch (error) {
     log.warn(`Failed to fetch versions: ${error}`);
     yield put(errorHandler.createErrorAction(error));
```